unstructured/metrics/table/table_eval.py

```diff
@@ -0,0 +1,215 @@
+            prediction = json.load(f)
+        with open(ground_truth_file) as f:
+            ground_truth = json.load(f)
+        if cutoff is not None:
+            return cls(prediction=prediction, ground_truth=ground_truth, cutoff=cutoff)
+        else:
+            return cls(prediction=prediction, ground_truth=ground_truth)
+
+    def process_file(self) -> TableEvaluation:
+        """Processes the files and computes table-level and element-level accuracy.
+
+        Returns:
+            TableEvaluation: A dataclass object containing the computed metrics.
+        """
+        total_predicted_tables = 0
+        total_tables = 0
+        total_row_index_acc = []
+        total_col_index_acc = []
+        total_row_content_acc = []
+        total_col_content_acc = []
+
+        predicted_table_data = extract_and_convert_tables_from_prediction(
+            self.prediction,
+        )
+        ground_truth_table_data = extract_and_convert_tables_from_ground_truth(
+            self.ground_truth,
+        )
+
+        matched_indices = TableAlignment.get_table_level_alignment(
+            predicted_table_data,
+            ground_truth_table_data,
+        )
+        total_predicted_tables += _count_predicted_tables(matched_indices)
+        total_tables += len(ground_truth_table_data)
+
+        metrics = TableAlignment.get_element_level_alignment(
+            predicted_table_data,
+            ground_truth_table_data,
+            matched_indices,
+            cutoff=self.cutoff,
+        )
+        if metrics:
+            total_col_index_acc.append(metrics["col_index_acc"])
+            total_row_index_acc.append(metrics["row_index_acc"])
+            total_col_content_acc.append(metrics["col_content_acc"])
+            total_row_content_acc.append(metrics["row_content_acc"])
+
+        return TableEvaluation(
+            total_tables=total_tables,
+            table_level_acc=(
+                round(total_predicted_tables / total_tables, 2) if total_tables else -1.0
+            ),
+            element_col_level_index_acc=(
+                round(np.mean(total_col_index_acc), 2) if len(total_col_index_acc) > 0 else -1.0
+            ),
+            element_row_level_index_acc=(
+                round(np.mean(total_row_index_acc), 2) if len(total_row_index_acc) > 0 else -1.0
+            ),
+            element_col_level_content_acc=(
+                round(np.mean(total_col_content_acc), 2) if len(total_col_content_acc) > 0 else -1.0
+            ),
+            element_row_level_content_acc=(
+                round(np.mean(total_row_content_acc), 2) if len(total_row_content_acc) > 0 else -1.0
+            ),
+        )
+
+
+@click.command()
+@click.option(
+    "--prediction_file", help="Path to the model prediction JSON file", type=click.Path(exists=True)
+)
+@click.option(
+    "--ground_truth_file", help="Path to the ground truth JSON file", type=click.Path(exists=True)
+)
+def run(prediction_file: str, ground_truth_file: str):
```

Comment:
lets also add the cut off to click options here

Reply:
Good catch! Fixed :)